src/SparkConduit.sol

```diff
@@ -191,7 +181,7 @@ contract SparkConduit is UpgradeableProxied, ISparkConduit, IInterestRateDataSou
         uint256 withdrawals = requestedShares[asset][ilk];
```

Comment:
Rename to `requestedShares_` or something.

Reply:
Fixed